contracts/MockUSDC.sol

```diff
@@ -0,0 +1,18 @@
+pragma solidity ^0.5.13;
+
+import { ERC20 } from "lib/mento-core/lib/openzeppelin-contracts/contracts/token/ERC20/ERC20.sol";
+import { ERC20Detailed } from "lib/mento-core/lib/openzeppelin-contracts/contracts/token/ERC20/ERC20Detailed.sol";
+import { Ownable } from "lib/mento-core/lib/openzeppelin-contracts/contracts/ownership/Ownable.sol";
+
+
+contract MockUSDC is ERC20, ERC20Detailed, Ownable {
+
+  constructor() public ERC20Detailed("mockUSDC", "USDC", 18) {
```

Comment:
This is great, but I think we should do this slightly differently. Let's add a constructor that takes the name, symbol and decimals as parameters. Then rename this to MockERC20. It's likely that we'll have a need for additional ERC20 tokens we control for testing so this would make this a bit more generic.

Reply:
good point 👍 should be fixed now